src/myna/application/deer/creep_timeseries_region/app.py

```diff
@@ -0,0 +1,234 @@
+                ]
+                process = self.start_subprocess_with_MPI_args(
+                    cmd_args,
+                    stdout=f,
+                    stderr=subprocess.STDOUT,
+                )
+        return process
+
+    def run_all_cases(self):
+        """Run all cases for the Myna step"""
+        exodus_files = self.settings["data"]["output_paths"][self.last_step_name]
+        csv_files = self.settings["data"]["output_paths"][self.step_name]
+        processes = []
+        for exodus_file, csv_file in zip(exodus_files, csv_files):
+            if (not os.path.exists(csv_file)) or (self.args.overwrite):
+                # Execute the case
+                process = self.run_case(exodus_file, os.path.dirname(csv_file))
+
+                # Handle serial versus batch submission processes
+                if self.args.batch:
+                    processes.append(process)
+                else:
+                    returncode = process.wait()
+                    if returncode != 0:
+                        error_msg = (
+                            f"Subprocess exited with return code {returncode}."
+                            + " Check case log files for details."
+                        )
+                        raise subprocess.SubprocessError(error_msg)
+
+        # Wait for batched jobs to finish
+        if self.args.batch:
+            for process in processes:
+                returncode = process.wait()
+                if returncode != 0:
+                    error_msg = (
+                        f"Subprocess exited with return code {returncode}. "
+                        + "Check case log files for details."
+                    )
+                    raise subprocess.SubprocessError(error_msg)
+
+        # Copy output to the Myna format
+        for myna_file in csv_files:
+            case_dir = os.path.dirname(myna_file)
+            case_file = os.path.join(case_dir, self.output_csv_name)
+            shutil.copy(case_file, myna_file)
+
+    def generate_orientation_file(self, case_dir, exodus_mesh_file):
+        """Generates the orientation file for an Exodus file with stored Euler angle
+        variables: "euler_bunge_zxz_phi1", "euler_bunge_zxz_Phi", and "euler_bunge_zxz_phi2"
+
+        Args:
+            case_dir: path to case directory to generate the orientation file for
+            exodus_mesh_file: path to the mesh file to get orientations from
+        """
+        try:
+            from netCDF4 import Dataset  # pylint: disable=import-outside-toplevel
+        except ImportError as exc:
+            raise ImportError(
+                'Myna deer app requires "pip install .[netcdf]" optional dependencies!'
+            ) from exc
+
+        # Get max block number from mesh
+        block_num_max = get_mesh_max_block_num(exodus_mesh_file)
+
+        # Get orientation data from Exodus mesh variables
+        with Dataset(exodus_mesh_file) as mesh:
```

Comment:
This one could also be wrapped in the class, right? Presumably `data.write( "eb_prop1", "euler_bunge_zxz_phi1", ... )`